package.json

```diff
@@ -58,6 +58,7 @@
     "angular-calendar": "0.23.2",
     "angular-draggable-droppable": "^2.0.0",
     "angular-resizable-element": "^2.0.0",
+    "angular-svg-icon": "^5.1.1",
```

Comment:
Remove the ^ from this line, looking at angular-svg-icon it seems newer versions are meant for angular 6, the ^ could introduce a non-working version of this dependency.